tests/fixtures/sample-restrictions.json

```diff
@@ -1 +0,0 @@
-[{"name":"mock_post_order","version":1,"rawRestrictions":[{"SenderOwnsAllInputs":[]},{"SenderHasInputRole":[{"index":0,"roleKey":"Supplier"}]},{"op":"and"},{"FixedOutputMetadataValueType":[{"index":0,"metadataKey":"SomeMetadataKey","metadataValueType":"Literal"},{"index":0,"metadataKey":"SomeOtherMetadataKey","metadataValueType":"File"}]},{"op":"and"},{"op":"and"}]},{"name":"mock_accept_order","version":1,"rawRestrictions":[{"SenderOwnsAllInputs":[]},{"op":"or"},{"SenderHasInputRole":[{"index":0,"roleKey":"Supplier"}]}]}]
```

Comment:
Why has this been deleted?

Reply:
was not used.

Reply:
This is something I sed before binary operator implementation

Reply:
Marking as resolved, as this file was just a zombie file, even wrong format.